src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+public class Tagging {
+    
+    private static ArrayList<String> listOfTaggings;
+    
+    private boolean isAdd;
+    private String personName;
```

Comment:
I will tag the right person with important information such as contact number/email as no multiple people can have the same contact number and email.